tests/components/autocomplete_dropdown.test.ts

```diff
@@ -312,6 +337,30 @@ describe("Autocomplete parenthesis", () => {
     expect(model.getters.getComposerSelection()).toEqual({ start: 5, end: 5 });
   });
 
+  test.each(["Enter", "Tab"])(
+    "=SUM( + edit SUM with autocomplete does not add left parenthesis",
```

Comment:
is it really what you were testing against? Was the bug allowing one to duplicate the left parenthesis?

Reply:
Not primarily, I was going through the test cases and I thought it's a good thing to add test cases for these two cases as well. We can remove it ofcourse if it's an extra.